Installation/cmake/modules/FindIPE.cmake

```
@@ -26,7 +26,7 @@ else()
 
   if(IPE_INCLUDE_DIR)
     file(READ "${IPE_INCLUDE_DIR}/ipebase.h" IPEBASE_H)
-    string(REGEX MATCH "IPELIB_VERSION[ ]*=[ ]*([67])([0-9][0-9])([0-9][0-9]);" FOUND_IPE_VERSION "${IPEBASE_H}")
+    string(REGEX MATCH "IPELIB_VERSION[ ]*=[ ]*([7])([0-9][0-9])([0-9][0-9]);" FOUND_IPE_VERSION "${IPEBASE_H}")
```

```suggestion
    string(REGEX MATCH "IPELIB_VERSION[ ]*=[ ]*([6789])([0-9][0-9])([0-9][0-9]);" FOUND_IPE_VERSION "${IPEBASE_H}")
```
For Ipe 6, and future Ipe 8, I think we still want to fill the variables `IPE_VERSION` and `IPE_FULL_VERSION`, even if we only support version 7, for the moment.